README.md

```diff
@@ -1,16 +1,34 @@
-# Title
+# Ruby Basics Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define the body of each method
 
 ## Introduction
 
-## SWBAT 1
+For this lab, you're going to be reviewing the basics of the Ruby language. Once
+you fork and clone the lab down onto your computer, run the test suite with the
+`learn` or `rspec` command. Read the test output in your terminal to get your
```

Comment:
"...output in your terminal and write the method to satisfy the test's expectations"

"Get tests to pass" is probably a bit too programmery for them.

ALSO: Have we taught 'coding to a test' yet?  Can you find out where that's taught? It's likely in another brick, but I want to know where it is.

Reply:
@sgharms looks like it's supposed to be covered or was to be covered in a "canceled" lesson in `Intro to Programming: Performing Basic Work in Ruby`, which comes after this brick in AT.

Reply:
Uhm. Awkward on me there.

Reply:
I think this needs to be extracted out somewhere.....like...how to pass tests is not a feature of "bundling work in Ruby." There needs to be some other place to stuff this lesson. I'm not sure where.